plugins/source/terraform/services/tfdata_test.go

```diff
@@ -0,0 +1,23 @@
+package services_test
+
+//import (
+//	"testing"
+//
+//	"github.com/cloudquery/cq-provider-sdk/provider"
+//	providertest "github.com/cloudquery/cq-provider-sdk/provider/testing"
+//)
+//
+//func TestTfData(t *testing.T) {
+//	cfg := `
+//    config:
+//      - name: mylocal
+//        local:
+//          path: "../examples/terraform.tfstate"
+//`
+//
+//	// TODO
+//	providertest.TestResource(t, providertest.ResourceTestCase{
+//		Provider: provider.Provider(),
+//		Config:   cfg,
+//	})
+//}
```

Comment:
Would it be possible to add this test back in now?

Reply:
I don't think we have this type of test in any other place, so I'm not sure how it will work (and if it will be accepted by @yevgenypats ). If you have a example plugin that does this already, I'm happy to port that code here.

Reply:
I think the terraform plugin is a bit special in this regard, because we want to test that it is able to read and decode a local terraform state file. I ported the test to do this using the new SDK in https://github.com/cloudquery/cloudquery/pull/1780/commits/730cdaa6188d1ea286bc0063190b00214470c4fd (pushed to this branch) - happy to hear @yevgenypats's thoughts here though

Reply:
I don't see a reason for that not to work - just as our mock tests are working.